packages/sdk/src/global-account/react/components/AccountAssets/AccountAssets.tsx

```diff
@@ -1,16 +1,55 @@
-import { SimpleHashNFTResponse } from "@b3dotfun/sdk/global-account/types/simplehash.types";
+import { NFT, SimpleHashNFTResponse } from "@b3dotfun/sdk/global-account/types/simplehash.types";
+import { useState } from "react";
 
 interface AccountAssetsProps {
   nfts: SimpleHashNFTResponse;
   isLoading?: boolean;
 }
 
+interface GroupedNFTs {
+  collection_id: string;
+  collection_name: string;
+  collection_image: string;
+  nfts: NFT[];
+}
+
 export function AccountAssets({ nfts, isLoading }: AccountAssetsProps) {
+  // Initialize with all collections expanded
+  // Group NFTs by collection
+  const groupedNFTs = nfts?.nfts?.reduce(
```

Comment:
Did we explore filtering both NFTs and tokens for spam? @aqt-dev mentioned a flag in some of the APIs?